doc/Contribution.md

```diff
@@ -0,0 +1,106 @@
+completed, a Pull Request will be created from the feature branch to `develop`. Remember to check both the linters
+and the tests before creating the Pull Request.
+
+Bug fixes work the same way as other tasks, with the exception of the branch name, that should be called `bug/<bugName>`.
+
+In order to contribute to the repository, these same scheme should be replicated in the forked repositories, so the
+new features or fixes should all come from the current version of `develop` and end up in `develop` again.
+
+All the `task/*` and `bug/*` branches are temporary, and should be removed once they have been merged.
+
+There is another set of branches called `release/<versionNumber>`, one for each version of the product. This branches
+point to each of the released versions of the project, they are permanent and they are created with each release.
+
+## Changelog
+The project contains a version changelog, called CHANGES_NEXT_RELEASE, that can be found in the root of the project.
+Whenever a new feature or bug fix is going to be merged with `develop`, a new entry should be added to this changelog.
+The new entry should contain the reference number of the issue it is solving (if any).
+
+When a new version is released, the changelog is cleared, and remains fixed in the last commit of that version. The
+content of the changelog is also moved to the release description in the Github release.
+
+## Releasing
+The process of making a release consists of the following steps:
+1. Create a new task branch changing the development version number in the package.json (with a sufix `-next`), to the
+new target version (without any sufix), and PR into `develop`.
+2. Create a tag from the last version of `develop` named with the version number and push it to the repository.
+3. Create the release in Github, from the created tag. In the description, add the contents of the Changelog.
+4. Create a release branch from the last version of `develop` named with the version number.
+5. PR `develop` into `master`.
+6. Create a new task for preparing the next release, adding the sufix `-next` to the current version number (to signal
+this as the development version).
+
+## Version numbers
+The version number will change for each release, according to the following rules:
+
+* All version numbers will always follow the common pattern: `X.Y.Z`
+* *X* will change only when there are changes in the release breaking backwards compatibility, or when there are
+very important changes in the feature set of the component. If X changes, Y is set to 0.
+* *Y* will change every time a new version is released. If only Y changes, it means some new features or bug fixes
+have been released, but the component is just an improved version of the current major release.
+* *Z* will be reserved for bugfixes inside the releases.
+
+Between releases, the version number will be `X.Y.Z-next` (where `X.Y.Z` is the last stable release), indicating
```

Comment:
Between releases, the version number will ... -> Between releases, the version number in develop branch will ...

(Maybe more precise)